necroHeal.lic

```diff
@@ -0,0 +1,57 @@
+=begin
+
+  Documentation: https://elanthipedia.play.net/Lich_script_development#necroheal
+  ## ** Placeming comments here until finalized and added to pedia.
+
+  Necromancer stand-alone healing script
+  
+  Uses all settings defined in HEAL YAML FILE (default)
+  Consume logic via combat trainer
+  
+  TO DO:
+  - See pull request #1462 for harvested materials.
+  - See if there is a background way to use common-healing to determine wound status
+  - Add YAML variable to swap YAML file name
+  
+=end
+
+custom_require.call(%w(common common-healing events))
+
+
+  if DRCH.check_health.empty?
+    echo 'Not wounded, exiting heal script'
+    exit
+  end
+
+
+  ## Sets flag to check for game line to determine when no wounds exist
+  ## Flag uses consume logic from combat-training. If no wounds are left
+  ## then harvest or dissection is used. Flags trigger off these rituals.
+  Flags.add('noWounds', 'You carefully position the corpse and make a cut', 'You bend over .* to make a few quick')

```

Comment:
Is this assuming the user has harvest or dissect as their training ritual? I can see some rare case occurring where the user might not be using these.